cpp/arcticdb/log/log.cpp

```diff
@@ -72,7 +72,7 @@ struct Loggers::Impl
 
 
 constexpr auto get_default_log_level() {
-    return spdlog::level::info;
+    return spdlog::level::trace;
```

Comment:
This needs backing out

Reply:
Oops, good spot

Reply:
Keep meaning to write a test for this 'cos I do it all the time